moped-editor/src/components/GridTable/Filters.js

```diff
@@ -763,18 +779,17 @@ const Filters = ({
           </Grid>
         </Hidden>
         <Grid item xs={12} md={2}>
-          <Grow in={handleApplyValidation() === null}>
```

Comment:
the search button appearing and disappearing kinda threw me off so i replaced this behavior in favor of disabling the search button if there are errors with the filter validation

Reply:
+1 - not seeing that button appear and disappear is ✨